mautic/mautic
# Fix Pipedrive mapping tab

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
This PR get back ***** placeholder for secret keys of integrations based on PluginBundle
Also fixed hide contact/company mapping after click on apply button.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Set up Pipedrive integration https://docs.mautic.org/en/plugins/pipedrive
3. After PR see placeholder are ****** for secret fields (token, password)
![image](https://user-images.githubusercontent.com/462477/112334430-b5b14a00-8cbb-11eb-8d98-7f519d9bd7d4.png)
4. After success set up, click on apply
5. Before PR mapping tabs should hide after apply, after PR works properly





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9820"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
